add: alert component

Shubhamkr2610 opened the pull request:
build alert component with side bar

Shubhamkr2610 committed:
add: alert component

build alert component with side bar

itsmebipul00 commented on components/alert/alert.css:
```diff
@@ -0,0 +1,54 @@
+}
+.alert{
+    text-align: center;
+    padding: 0px 20px 0px 20px;
```
use rems

Shubhamkr2610 replied:
I have  made the changes . Please check

varun13169 commented on components/alert/alert.css:
```diff
@@ -0,0 +1,54 @@
+@import url("/index.css");
```
For local files following will work.
@import "/index.css";

Not sure if using URL will have a performance impact or not, so please read about it.

Shubhamkr2610 replied:
Okay . I'll read about it and update in what will correct way to import file.

varun13169 commented on components/alert/alert.css:
```diff
@@ -0,0 +1,54 @@
+
+*{
+    margin:0;
+    padding: 0;
```
0 bananas ??? or 0 lightyear ???

mention units.

Shubhamkr2610 replied:
Should I use px or rem unit?

Shubhamkr2610 replied:
I have made the changes using px. Can you give your feedback . Is it right or not to use px?

varun13169 commented on components/alert/alert.css:
```diff
@@ -0,0 +1,54 @@
+.alert-sidebar-main-container{
+    margin: 50px 0px 0px 0px;
+    display: flex;
+}
```
use rem, please.

50px = 50/16 rem ~ 3.12rem

Shubhamkr2610 replied:
Changed to rem. Please review it

developer-anand commented on components/alert/sidebar.css:
```diff
@@ -0,0 +1,40 @@
+*{
```
you can put it in your index css so you don't have to write it again and again.

Shubhamkr2610 replied:
Okay . I'll read about it and update in what will correct way to import file.

Shubhamkr2610 replied:
Moved it  to index.css Please look into it

varun13169 commented on index.css:
```diff
@@ -31,6 +41,8 @@ body {
     margin-left: 1100px;
     margin-top: -60px;
+    
+    
```
why this is part of reset CSS ???

Shubhamkr2610 committed:
changed px to rem

Made the changes (px to rem).  Also moved  universal selector (*)  to parent CSS file so that I will not have use again and again

humzahasan commented on components/alert/sidebar.css:
```diff
@@ -0,0 +1,33 @@
+}
+
+h3{
+    font-size: 35px;
```
Can this be refactored to rem ?

Shubhamkr2610 replied:
yes.. of-course

Shubhamkr2610 replied:
made the changes . please review

Shubhamkr2610 replied:
removed the commented and unused code. please review .

humzahasan reviewed (approved):
Few of the mesurments are in px, refacor to rem to maintain consistency

Shubhamkr2610 committed:
refactor: px to rem

Muzammil-khan710 reviewed (commented):
Remove comment code from index.css file..

arjun-computer-geek commented on components/alert/alert.html:
```diff
@@ -0,0 +1,130 @@
+
+        <h2>Made with ❤️ by shubham</p>
+            <h3>connect with me 👇 </p>
+                <a href="https://github.com/Shubhamkr2610">github<i class="fab fa-github-square"></i></a>
```
Always use target=_blank with the anchor attributes for external links

Shubhamkr2610 replied:
I implemented this to my code. please check and review it

arjun-computer-geek commented on components/alert/sidebar.css:
```diff
@@ -0,0 +1,32 @@
+.aside-left{
+    width: 30%;
+    height: 100vh;
+    /* padding: 20px; */
```
Always remove unused code

Shubhamkr2610 replied:
removed . please review it.

arjun-computer-geek reviewed (changes requested):
Do some changes

Shubhamkr2610 committed:
refactor: removed unused code

Shubhamkr2610 commented:
> 



> Remove comment code from index.css file..

okay, removed . please review it.

Shubhamkr2610 committed:
refactor: used target attribute

developer-anand reviewed (approved):
Good to go.

arjun-computer-geek reviewed (approved):
Now looks good to me

Shubhamkr2610 committed:
added:  target attribute
